content/common/cli-reference.md

```diff
@@ -1238,17 +1314,34 @@ CLI commands have four phases that indicate levels of stability:
 
 {{ startTab "v1.0.0"}}
 
-| Command                                                    | Stability   |
-| ---------------------------------------------------------- | ----------- |
-| <code>spin add</code>                                                 | Stable      |
-| <code>spin build</code>                                               | Stable      |
-| <code>spin new</code>                                                 | Stable      |
+| Command                                                                               | Stability   |
```

Comment:
What has changed on these lines?  If just reformatting maybe revert so this PR is clean?